gradle/version.properties

```diff
@@ -8,7 +8,7 @@ commonslang3=3.5
 commonscollections=3.2.2
 netty4=4.1.36.Final
 jsr305=3.0.1
-aws=1.11.223
+aws=1.11.575
```

Comment:
latest one is 1.11.576, any reason not to use that version?

Reply:
i checked it on `mvnrepository.com` an hour ago and the latest was `1.11.575`, it seems, it just got there :) will update.